examples/tool_chat_template_llama3_json.jinja

```diff
@@ -3,7 +3,7 @@
     {%- set tools = custom_tools %}
 {%- endif %}
 {%- if not tools_in_user_message is defined %}
-    {%- set tools_in_user_message = false %}
+    {%- set tools_in_user_message = true %}
```

Comment:
for what it's worth I have had great success with keeping tools in system message, but that's just me